ci/build-pbs-packages.sh

```diff
@@ -5,42 +5,40 @@
 pbsdir=/pbssrc
 rpm_dir=/root/rpmbuild
 
+rm -rf /src/packages
 mkdir -p /src/packages
 mkdir -p ${rpm_dir}/{BUILD,RPMS,SOURCES,SPECS,SRPMS}
 
 cp -r $pbsdir /tmp/pbs
 cd /tmp/pbs
 ./autogen.sh
-cflags="-g -O2 -Wall -Werror"
-if [ "x${ID}" == "xdebian" -o "x${ID}" == "xubuntu" ]; then
-	cflags="${cflags} -Wno-unused-result"
-fi
-./configure CFLAGS="${cflags}" --prefix=/opt/pbs --enable-ptl
+mkdir target
+cd target
+../configure --prefix=/opt/pbs --enable-ptl
```

Comment:
Don't we need <cflags="-g -O2 -Wall -Werror"> here.

Reply:
No, here we run configure only for "make dist", rpmbuild will run configure again and at that time we need CFLAGS.